descriptors/ModuleDescriptor-template.json

```diff
@@ -255,7 +255,11 @@
             "acquisitions-units-storage.memberships.collection.get",
             "isbn-utils.convert-to-13.get",
             "instance-authority-links.instances.collection.get",
-            "instance-authority-links.instances.collection.put"
+            "instance-authority-links.instances.collection.put",
+            "user-tenants.collection.get",
+            "organizations.organizations.collection.get",
+            "invoices.acquisitions-units-assignments.assign",
```

Comment:
as far as I know, we should not add this permission here, since this permission is optional and a user assigned to a particular acquisition unit should be provided with it

Reply:
fixed

Reply:
please, remove "user-tenants.collection.get" permission here as well since it is needed by consortium-related functionality that is not present in Orchid release